packages/pluggableWidgets/datagrid-text-filter-web/src/components/__tests__/DatagridTextFilter.spec.tsx

```diff
@@ -24,6 +25,56 @@ describe("Text Filter", () => {
             delete (global as any)["com.mendix.widgets.web.UUID"];
         });
 
+        /**
+         * Syncing defaultValue is bed practice in React,
+         * but this is how filter works right now, so we
+         * have to test and support this behaviour.
+         */
```

Comment:
I don't think this comment adds any value, please drop it.

Reply:
@r0b1n, done